src/ensembl/production/datacheck/app/main.py

```diff
@@ -225,7 +225,10 @@ def job_submit(payload=None):
     input_data = dict(payload)
 
     assert_mysql_uri(input_data['server_url'])
-    
+
+    if 'target_url' not in input_data:
+        input_data['target_url'] = input_data['server_url']
```

Comment:
```suggestion
    if 'target_url' in input_data:
        input_data['server_uri'] = input_data['target_url'].split(',')
```
There is no 'target_url' in the pipeline - it's called `server_uri`, but we don't call it that until now, to avoid confusion with the pre-existing, almost-identical, `server_url`. The pipeline is expecting a list, so even if there is just one value, which is the expected use case, use split to create a list with one element. If target_url is not given, we do not want to substitute server_url instead - that's already been used to set the registry, it's either redundant or would cause errors to do that.